cpi/zig/build.zig.zon

```diff
@@ -0,0 +1,46 @@
+.{
+    .name = "solana-program-rosetta-cpi-zig",
+    // This is a [Semantic Version](https://semver.org/).
+    // In a future version of Zig it will be used for package deduplication.
+    .version = "0.13.0",
+
+    // This field is optional.
+    // This is currently advisory only; Zig does not yet do anything
+    // with this value.
+    .minimum_zig_version = "0.13.0",
+
+    // This field is optional.
+    // Each dependency must either provide a `url` and `hash`, or a `path`.
+    // `zig build --fetch` can be used to fetch all dependencies of a package, recursively.
+    // Once all dependencies are fetched, `zig build` no longer requires
+    // internet connectivity.
+    .dependencies = .{
+        .base58 = .{
+            .url = "https://github.com/joncinque/base58-zig/archive/refs/tags/v0.13.3.tar.gz",
+            .hash = "1220fd067bf167b9062cc29ccf715ff97643c2d3f8958beea863b6036876bb71bcb8",
+        },
+        .@"solana-program-sdk" = .{
+            .url = "https://github.com/joncinque/solana-program-sdk-zig/archive/refs/tags/v0.13.1.tar.gz",
+            .hash = "122030336f1257e3c0aa64243f5243f554b903c6b9ef3a91d48bfbe896c0c7d9b13b",
+        },
+        .@"solana-program-library" = .{
+            .url = "https://github.com/sonicfromnewyoke/solana-program-library-zig/archive/1cb267876b7a48345c9a081587999ee685131e24.tar.gz",
```

Comment:
this import should be replaced with the original repo instead of my fork after merging https://github.com/joncinque/solana-program-library-zig/pull/3

Reply:
Feel free to update this, 0.13.1 is available now: https://github.com/joncinque/solana-program-library-zig/releases/tag/v0.13.1